resources/assets/uad_lists.json

```
@@ -18929,7 +18929,7 @@
   {
     "id": "com.samsung.android.contacts",
     "list": "Oem",
-    "description": "Samsung contacts app?",
+    "description": "Samsung Contacts App - It's safe to uninstall but you'll need to see for some alternatives like Simple Contacts, some contact apps doesn't works as intended and doesn't really saves contacts in device's storage.",
```

```suggestion
    "description": "Samsung Contacts App\nSafe to uninstall but you'll need to download a replacement contacts app. Some contact apps doesn't works as intended and doesn't really saves contacts in device's storage.",
```

I'd refrain from getting subjective, such as recommending an alternative.